apps/meteor/server/routes/avatar/utils.ts

```diff
@@ -18,6 +18,7 @@ const cookie = new Cookies();
 
 export const MAX_SVG_AVATAR_SIZE = 1024;
 export const MIN_SVG_AVATAR_SIZE = 16;
+const MAX_SVG_AVATAR_INITIALS = 3;
```

Comment:
It seems this is used as a count of how many initials we are using, do we support larger charsets?
I don't think this would work with japanese, or chineese characters (not sure if we support them tho)

Reply:
It looks like if you change the `getFirstLetter` to just be something like this:
```ts
const getFirstLetter = (name: string) => name.charAt(0);
```
Or even inline it, we can better handle unicode strings (I am just not 100% if they will be rendered properly in the icon)

Reply:
> It looks like if you change the `getFirstLetter` to just be something like this:
> I just think we need to sanitize name before fetching the first char, I think we use DOMPurify to do that right?

There's another PR that touches this function to support UTF8 chars, so we can skip this for now.

Reply:
> It seems this is used as a count of how many initials we are using, do we support larger 4-byte character points? I don't think this would work with japanese, or chinese characters (not sure if we support them tho)

Don't think we currently support them, but we can revisit that after we support at least 1 character svgs

Reply:
For sure :)
just wanted to bring to attention